integration_tests/test_basic.py

```diff
@@ -991,6 +992,41 @@ def test_replay_protection(cronos):
         w3.eth.send_raw_transaction(HexBytes(raw))
 
 
+@pytest.fixture(scope="module")
```

Comment:
nitty picky but may be cleaner to move to another file since it is using a different instance of cronos